src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseAgreedSettledPartAdmitDefendantLipNotificationHandlerTest.java

```diff
@@ -105,6 +108,50 @@ void shouldNotNotify_whenInvoked_spec_lip_noEmail() {
             verify(notificationService, never()).sendMail(any(), any(), any(), any());
         }
 
+        @Test
+        void shouldNotNotify_whenInvoked_spec_lip_bilingual() {
+            CaseData caseData = CaseDataBuilder.builder()
+                .atStateClaimDetailsNotified()
+                .specClaim1v1LrVsLip()
+                .caseDataLip(CaseDataLiP.builder().respondent1LiPResponse(RespondentLiPResponse.builder().respondent1ResponseLanguage("BOTH").build()).build())
+                .build();
+
+            CallbackParams params = CallbackParamsBuilder.builder().of(ABOUT_TO_SUBMIT, caseData).request(
+                CallbackRequest.builder().eventId("NOTIFY_LIP_DEFENDANT_PART_ADMIT_CLAIM_SETTLED")
+                    .build()).build();
+
+            handler.handle(params);
+
+            verify(notificationService).sendMail(
+                "sole.trader@email.com",
```

Comment:
can be created string const for all static strings.

Reply:
Updated